service.bal

```diff
@@ -75,7 +75,10 @@ service /admin on new http:Listener(9090) {
 
     # A resource for creating an Aid-Package
     # + return - An Aid-Package
-    resource function post aidpackages(@http:Payload AidPackage aidPackage) returns AidPackage|error {
+    resource function post aidpackages(@http:Payload AidPackage aidPackage, @http:Header {name: "X-User-Name"} string createdBy) returns AidPackage|error {
```

Comment:
What is `X-User-Name` ? is this something send by Choreo?

Reply:
Thought to use this custom header to retrieve the relevant information. But, I think we don't need it anymore.

Reply:
Fixed with [226dd707491404c8faf0e33dd835a3f64c789a57](https://github.com/LSFLK/MedicinesforLK-AdminAPI/pull/68/commits/226dd707491404c8faf0e33dd835a3f64c789a57)